cpp/velox/compute/VeloxColumnarToRowConverter.cc

```diff
@@ -254,45 +257,36 @@ arrow::Status VeloxColumnarToRowConverter::FillBuffer(
           break;
         }
         case arrow::Decimal128Type::type_id: {
-          /*auto out_array = dynamic_cast<arrow::Decimal128Array*>(array.get());
-          auto dtype = dynamic_cast<arrow::Decimal128Type*>(out_array->type().get());
+          for (auto j = 0; j < num_rows_; j++) {
```

Comment:
I am not very familiar with avx instructions here, please help me to add avx optimization in decimal support.